ibm/mas_devops/roles/suite_app_config/vars/defaultspecs/predict.yml

```diff
@@ -7,6 +7,5 @@ mas_app_ws_spec:
   settings:
     wml:
       instance_id: "{{ lookup('env', 'WML_INSTANCE_ID') | default('openshift', true) }}"
-      url: >-
-        {{ cpd_url }}
-      version: "{{ cpd_product_version }}"
+      url: "{{ lookup('env', 'WML_URL') | default('https://internal-nginx-svc.ibm-cpd.svc:12443', true) }}"
```

Comment:
The lookups should all live in the defaults/main.yml file of the role, and just reference them vars here.

Reply:
fixed, moved lookup to defaults/predict.yml